### PR TITLE
[DRFT-147] baseline fact modal

### PR DESCRIPTION
**Pull request by opacut**
- Add and Delete modals for baseline facts
- TextInput elements
- buttons

### CODE REVIEW

File: src/SmartComponents/BaselinesPage/EditBaseline/DeleteFactModal/DeleteFactModal.js [hunk 1]
```diff
@@ -31,20 +31,23 @@ class DeleteFactModal extends Component {
         return (
             <Modal
                 className="drift"
+                ouiaId='delete-fact-modal'
```
Comment (johnsonm325) [inline src/SmartComponents/BaselinesPage/EditBaseline/DeleteFactModal/DeleteFactModal.js:34]: Should we make this consistent? `delete-fact` or `delete-facts` for both.

File: src/SmartComponents/BaselinesPage/EditBaseline/DeleteFactModal/DeleteFactModal.js [hunk 2]
```diff
@@ -31,20 +31,23 @@ class DeleteFactModal extends Component {
...
                         key="confirm"
+                        ouiaId='delete-facts-button'
                         variant="danger"
                         onClick={ deleteFact ? deleteFact : this.deleteFacts }
                     >
                     Delete facts
                     </Button>,
                     <Button
                         key="cancel"
+                        ouiaId='cancel-button'
```
Comment (johnsonm325) [inline src/SmartComponents/BaselinesPage/EditBaseline/DeleteFactModal/DeleteFactModal.js:50]: Should we specify what cancel button this is for? `delete-fact-cancel-button`

**Review by johnsonm325 — CHANGES_REQUESTED** [review]

**Comment by opacut** [comment]
ammended and rebased

File: src/SmartComponents/BaselinesPage/EditBaseline/FactModal/FactModal.js [hunk 3]
```diff
@@ -225,14 +232,14 @@ export class FactModal extends Component {
                     <Button
                         key="confirm"
                         variant="primary"
-                        ouiaId="save"
+                        ouiaId="save-button"
```
Comment (johnsonm325) [inline src/SmartComponents/BaselinesPage/EditBaseline/FactModal/FactModal.js:235]: I should have asked the same question here. This goes for every button like this. Though, `save-button` may be unique. Do we want to call it `fact-modal-save-button`? It's your call really.

File: src/SmartComponents/BaselinesPage/EditBaseline/FactModal/FactModal.js [hunk 4]
```diff
@@ -225,14 +232,14 @@ export class FactModal extends Component {
...
                         onClick={ this.confirmModal }>
                         Save
                     </Button>,
                     <Button
                         key="cancel"
                         variant="link"
-                        ouiaId="cancel"
+                        ouiaId="cancel-button"
```
Comment (johnsonm325) [inline src/SmartComponents/BaselinesPage/EditBaseline/FactModal/FactModal.js:242]: `fact-modal-cancel-button`?

**Review by johnsonm325 — CHANGES_REQUESTED** [review]

**Comment by opacut** [comment]
changed and rebased

**Review by johnsonm325 — APPROVED** [review]

**Pull request merged**